infra/base-images/base-builder/jcc/jcc.go

```diff
@@ -324,12 +339,21 @@ func main() {
 	} else {
 		bin = "clang"
 	}
-	fullCmdArgs := append([]string{bin}, newArgs...)
-	retcode, out, errstr := Compile(bin, newArgs)
-	if retcode == 0 {
-		WriteStdErrOut(fullCmdArgs, out, errstr)
-		os.Exit(0)
-	}
+ 	fullCmdArgs := append([]string{bin}, newArgs...)
+ 	retcode, out, errstr := Compile(bin, newArgs)
+ 	if retcode == 0 {
+ 	 	WriteStdErrOut(fullCmdArgs, out, errstr)
+ 	 	os.Exit(0)
+ 	}
```

Comment:
These lines are identical except for their indentations.
The original lines are misaligned with others, likely due to space/tab.